modulemd/v2/tests/test-modulemd-modulestream.c

```diff
@@ -616,6 +616,54 @@ module_stream_v2_test_depends_on_stream (ModuleStreamFixture *fixture,
   g_clear_object (&stream);
 }
 
+static void
+module_stream_test_community (ModuleStreamFixture *fixture,
+                                         gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleStream) stream = NULL;
+  guint64 version;
+  g_autofree gchar *community = NULL;
+  
+  for (version = MD_MODULESTREAM_VERSION_ONE;
+       version <= MD_MODULESTREAM_VERSION_LATEST;
+       version++)
+  {
+    stream = modulemd_module_stream_new (version, "foo", "latest");
+    g_assert_nonnull (stream);
+    
+    g_assert_null (modulemd_module_stream_get_community(stream));
```

Comment:
`modulemd_module_stream_get_community()` doesn't exist. It needs to be `modulemd_module_stream_v1_get_community()` or `modulemd_module_stream_v2_get_community()`